src/usage/indexing.md

```diff
@@ -150,3 +59,29 @@ WITH (options = $$
 segment.max_growing_segment_size = 200000
 $$);
 ```
+
+## Suggestions
+
+Choosing the right indexing algorithm and options is important for the performance of the vector search. Here are some suggestions for different scenarios.
+
+### Brute force(FLAT)
+
+If you only perform a few searches(say 1000-10000) or you need guaranteed exact results, you can use the [brute force](/reference/indexing_options.html#options-for-table-flat) algorithm.
```

Comment:
"a few searches" is ambiguous. It depends on the candidate size, not the query size.

Reply:
This means that if you run only a few queries, the time spent building indexes won't be offset by faster search times.